entrypoint_scripts/test/travis-integration-test.sh

```diff
@@ -68,6 +68,8 @@ travis_fold start travis_integration_tests
 
 python tests/check_status.py -v
 python tests/smoke_test.py
+python tests/product_selenium_unittest.py
```

Comment:
Hold off on this for now. Everything will be integrated once all tests have been completed.

Reply:
alright it has been removed.